setup.js

```
@@ -0,0 +1,6 @@
+import 'dotenv/config';
```

Loading dotenv with `import 'dotenv/config'` is fine to populate process.env. Consider validating required environment variables early and failing fast with a helpful error if any required vars (DB_HOST, DB_USER, etc.) are missing.